### ISSUE TITLE
bug: 4.x - Logical CSS properties + overriding component classes

### ISSUE THREAD

**Issue by robsontenorio** [saadeghi/daisyui#2499]
### What version of daisyUI are you using?

4.0.1

### Which browsers are you seeing the problem on?

Chrome, Safari

### Reproduction URL

https://play.tailwindcss.com/u8LHdZbDsj

### Describe your issue

Is this expected?

```html
            // Works with 3.x
            <input class="input input-primary rounded-l-none" />

            // On 4.x  need to force with `!`
            <input class="input input-primary !rounded-l-none" />
```

<img width="791" alt="image" src="https://github.com/saadeghi/daisyui/assets/118955/2a9cafe3-5704-4ffe-ad66-05a549ba29db">

**Comment by robsontenorio** [comment]
Same for other things like background classes

```html
<select class="select select-primary bg-base-200"></select>
```

**Comment by nabilfatih** [comment]
same here, I think for most custom parts.. is this expected or bugs?

**Comment by saadeghi** [comment]
Thanks for reporting!  
Fixed in `4.0.2`
